packages/headless/src/controllers/triggers/headless-execute-trigger.ts

```diff
@@ -7,8 +7,10 @@ import {arrayEqual} from '../../utils/compare-utils';
 import {loadReducerError} from '../../utils/errors';
 import {buildController, Controller} from '../controller/headless-controller';
 
-/**
- * The `ExecuteTrigger` controller handles execute trigger actions.
+/** 
+ * The `Execute` triggers executes a custom JavaScript function call. The `ExecuteTrigger` controller handles execute triggers.
+ * 
+ * See [Trigger - Query pipeline feature](https://docs.coveo.com/en/1458/) and [Execute trigger reference](https://docs.coveo.com/en/3413/tune-relevance/manage-trigger-rules#execute).
```

Comment:
```suggestion
```

(I wouldn't include the link to /1458/; most people will configure those rules from the admin UI, so the link to 3413 is sufficient IMO)